scripts/prep-dev

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Developer-oriented utility script for deploying Saltstack config
+# files for the SecureDrop Workstation dev env. Installs the latest
+# locally built RPM in order to deploy the Salt config.
+set -e
+set -u
+set -o pipefail
+
+
+# The dest directory in dom0 is not customizable.
+dom0_dev_dir="$HOME/securedrop-workstation"
+
+function find_latest_rpm() {
+    find "${dom0_dev_dir}/rpm-build/RPMS/" -type f -iname '*.rpm' | sort -n | tail -n 1
+}
+
+latest_rpm="$(find_latest_rpm)"
+if [[ -z "$latest_rpm" ]]; then
+    echo "Could not find RPM!"
+    exit 1
+fi
+
+echo "Deploying Salt config..."
+echo "Installing RPM at $latest_rpm ..."
+sudo dnf install -y "$latest_rpm"
+
+# Always copy secrets, these won't be configured as part
+# of the RPM installation.
```

Comment:
Maybe add an `echo` for these operations as well, to make it clear that these files have been copied to a new destination?

Reply:
Great suggestion, added!